app/controllers/users_controller.rb

```diff
@@ -40,6 +40,20 @@ def create
       end
     end 
   end
+
+  def following
+    @title = "Following"
+    @user  = User.find(params[:id])
+    @users = @user.following
+    render 'show_follow'
```

Comment:
If it's not too much work it would be nice to just name it the default. I think if you just remove that render line it will throw an exception with the expected name, then you can just rename your view.